.github/workflows/coding-standards.yml

```diff
@@ -0,0 +1,41 @@
+
+name: "Coding Standards"
+
+on:
+  workflow_call:
+    inputs:
+      php-version:
+        description: "The PHP version to use when running the job"
+        default: "8.0"
+        required: false
```

Comment:
Not having this as mandatory means that changing that variable will be a BC-break.
I think we can create versions for this repo and reference it with "@v1", etc. though, so it should not be an issue.